packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.tsx

```diff
@@ -141,17 +146,19 @@ const FeaturedResearchTopCard = ({
     )
   }
   return (
-    <div>
-      <CardMedia
-        component="img"
-        image={url}
-        aria-hidden="true"
-        sx={{
-          objectFit: 'cover',
-          borderRadius: '10px',
-          marginBottom: '30px',
-        }}
-      />
+    <Box ref={ref}>
+      <Grow in={inView} timeout={transitionTimeoutMs}>
```

Comment:
I personally think `Grow` is pretty jarring and distracting here compared to `Fade`, but I defer to design